aggregator/src/aggregator.rs

```diff
@@ -3226,7 +3226,10 @@ impl VdafOps {
                         })?;
                     Q::acknowledge_collection(tx, task.id(), collection_job.batch_identifier())
                         .await?;
-                    Ok(collection_job)
+                    let spanned_interval = tx
+                        .get_interval_spanned_by_collection(&collection_job_id)
```

Comment:
I don't think we can do `acknowledge_collection` in parallel with the other two queries because we need the collection job to get the batch identifier, but I put `get_collection_job` and `get_interval_spanned_by_collection` into a `try_join`.

I considered adding a column for the spanned interval to `collection_jobs`, but I decided that a change that doesn't require changing the schema is simpler, and also we can avoid duplicating information in the database. I also figure that GET on collection jobs will happen far, far less often than hot paths like uploads or aggregation jobs, so my intuition is that it shouldn't be too performance critical. However we could record the spanned interval at the time of creating a collection job, if we find that this query is a problem.

Reply:
Ah, I didn't mean to suggest introducing a new field in the schema -- I meant to add an optional interval to the `Collection` model, filling it in "artificially" using logic like that currently written in `get_interval_by_collection`. Still -- this is fine, I strongly suspect pipelining the reads will lead to this being the ~same performance-wise.

Reply:
I see what you mean now. I did consider adding that field to `datastore::models::Collection`, but then you end up in this awkward spot: either you make it a required part of the struct and have `Datastore::get_collection_job` work out the interval, but then you force a `JOIN` across aggregation jobs on all of `get_collection_job`'s many callsites, when only a single one (the collection job GET handler) actually needs the spanned interval. Or you could make it an `Option`, but again, that's awkward since only a single code path cares about that option.